citesphere/pom.xml

```diff
@@ -25,7 +25,7 @@
         <spring.kafka.version>2.2.6.RELEASE</spring.kafka.version>
         <spring-social-zotero.version>0.7</spring-social-zotero.version>
         <citesphere.messages.version>0.4</citesphere.messages.version>
-        <citesphere.model.version>1.8</citesphere.model.version>
+        <citesphere.model.version>1.9-SNAPSHOT</citesphere.model.version>
```

Comment:
update `citesphere.model.version` once Julia releases a new version for `citesphere-model`. Moreover, the version number should be same as the official release version that's in maven central, not the one from citesphere-model `pom.xml` file. Please check that as well.